### PR TITLE
🔨 Switch Search to use useOvermind

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Follow-up of #2780 & #2819

### CODE REVIEW

File: packages/app/src/app/pages/Search/index.tsx [hunk 1]
```diff
@@ -3,46 +3,50 @@ import {
...
 
 import 'instantsearch.css/themes/reset.css';
 
-import { Container, Content, Main, StyledTitle } from './elements';
+import { Container, Content, Main, Title } from './elements';
 import Filters from './Filters';
 import Results from './Results';
 import Styles from './search';
 
-interface ISearchProps {
```
Comment (armujahid) [inline packages/app/src/app/pages/Search/index.tsx]: We Prefer interface over type for typing Props as mentioned in "Best Practices and Style Guide" section. So `ISearchProps` interface was fine

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2852/commits/0198b527680ca6a44f9414d9a297534beb4fa1b9 is at https://pr2852.build.csb.dev/s/new.

File: packages/app/src/app/pages/Search/index.tsx (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Search/index.tsx]: Yeah, why was this deleted?

File: packages/app/src/app/pages/Search/index.tsx [hunk 2]
```diff
@@ -3,46 +3,47 @@ import {
...
 const updateAfter = 700;
 
 const createURL = state => `?${qs.stringify(state)}`;
 
 const searchStateToUrl = (location, searchState) =>
   searchState ? `${location.pathname}${createURL(searchState)}` : '';
 
-const Search: React.FC<ISearchProps> = ({ history, location }) => {
+type Props = RouteComponentProps;
+const Search: FunctionComponent<Props> = ({ history, location }) => {
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Search/index.tsx]: @SaraVieira Props are still present.

**Comment by MichaelDeBoey** [comment]
Ping @CompuIves @SaraVieira @christianalfoni

**Comment by christianalfoni** [comment]
mmm... in `master` this is already refactored? Sorry 😭 Closing now, please reopen if I misunderstood something

**Pull request closed without merging**